test/integrationtests.sh

```diff
@@ -7,7 +7,7 @@ mkdir coverage/
 
 export GITREPO_TEMPLATES="git://gitops.sunet.se/cnaas-lab-templates"
 export GITREPO_SETTINGS="git://gitops.sunet.se/cnaas-lab-settings"
-export GITREPO_ETC="git://gitops.sunet.se/cnaas-lab-etc"
+export GITREPO_ETC="https://github.com/indy-independence/cnaas-nms-lab-etc.git"
```

Comment:
You sure about this?

Reply:
Maybe we could make it a bit more official by having etc/settings/templates repos for integration test env under SUNETs github, but I couldn't make it work with the gitops repo because of some IP that changed